### ISSUE TITLE
Update to MC Version 1.19.4

### ISSUE THREAD

**Issue by LJSkinner** [EmpressValla/TheEmeraldLasso#18]
This is still a long way away and in pre-release but when it is released and the dependencies are updated, the mod will be updated to the most recent version.

**Comment by LJSkinner** [comment]
Forge updated, still waiting on parchment mappings for 1.19.4.

**Comment by LJSkinner** [comment]
PR for 1.19.4 parchment mappings is on hold. I will update to version 1.19.4 as soon as this is available.

**Comment by LJSkinner** [comment]
I'm still waiting on the release channel for this. After this is done, I will update to 1.19.4 and also investigate #23. I will prioritise most recent versions (1.19.4, 1.19.2 and 1.18.2). I'm including 1.19.2 because I know there are still a lot of modpacks on this version. As per usual with any update, if you have an existing world just be careful as Minecraft can be screwy sometimes. Refer to #15 for potential solutions with left click.
